modules/ding_availability/ding_availability.module

```diff
@@ -251,6 +251,10 @@ function ding_availability_holdings($provider_ids) {
           'There are @count users in queue to loan the material.');
       }
 
+      if ($total_and_ordered_count == 0) {
+        ddbasic_body_class('no-holdings');
```

Comment:
Don't use DDBasic functions in backend modules as this will prevent libraries not using DDBasic to use DDB CMS and this module. E.g. ereolen, ereolenGo, Icelandic libraries etc.